src/ceph-volume/ceph_volume/util/device.py

```diff
@@ -386,9 +396,21 @@ def is_device(self):
         return False
 
     @property
-    def is_acceptable_device(self):
+    def is_crypt(self):
+        if self.disk_api:
+            return self.disk_api['TYPE'] == 'crypt'
+        elif self.blkid_api:
+            return self.blkid_api['TYPE'] == 'crypt'
+        return False
```

Comment:
Any reason to not use `is_encrypted`?

Reply:
IIRC, `is_encrypted` is for encrypted-by-Ceph devices.